INSTALL

```diff
@@ -806,7 +806,7 @@
                 possible to create your own ".conf" and ".tmpl" files and store
                 them locally, outside the OpenSSL source tree. This environment
                 variable can be set to the directory where these files are held
-                and will have Configure to consider them in addition to the
+                and will have Configure to consider them in preference to the
```

Comment:
I don't feel particularly strongly, but "in addition to", to me, indicates that it is appended at the end, so that if something is in the systemwide configuration it takes effect first.  Maybe we want something about "override".